modules/sfp_haveibeenpwned.py

```diff
@@ -135,23 +179,54 @@ def handleEvent(self, event):
             self.results[eventData] = True
 
         data = self.query(eventData)
-        if data is None:
+        if data is not None:
+            for n in data:
+                try:
+                    if not self.opts['api_key']:
+                        site = n["Title"]
+                    else:
+                        site = n["Name"]
+                except Exception as e:
+                    self.sf.debug(f"Unable to parse result from HaveIBeenPwned?: {e}")
+                    continue
+
+                evt = eventName + "_COMPROMISED"
```

Comment:
`eventName` will always be `EMAILADDR`, right? Better to be explicit, if that's the case.

Reply:
This was there from the beginning, maybe some different logic back then?